server/src/test/java/org/jboss/as/server/ServerEnvironmentTestCase.java

```diff
@@ -91,4 +92,12 @@ public void testUUIDLifeCycle() throws IOException {
         assertThat(uuids.get(0), is(not(uuid)));
         Files.delete(uuidPath);
     }
+
+    @Test
+    public void testAliasFunctionality() {
+        assertThat(ServerEnvironment.translateFileAlias("lb", Stability.DEFAULT), is("lb"));
+        assertThat(ServerEnvironment.translateFileAlias(null, Stability.DEFAULT), is((String) null));
+        assertThat(ServerEnvironment.translateFileAlias("lb", Stability.COMMUNITY), is("standalone-load-balancer.xml"));
+        assertThat(ServerEnvironment.translateFileAlias("custom.xml", Stability.COMMUNITY), is("custom.xml"));
+    }
```

Comment:
Looks good, but I wonder if we should test the functionality using a ServerEnvironent instance instead of the static method.

ServerEnvironent has a getServerConfigurationFile() method that can be used to assert the final file that will be in use.

Reply:
That would increase the reliability of the change, and the testUUIDLifeCycle() test has all the pieces required to initialize a ServerEnvironment, so looks very simple to add it.